uktrade/data-hub-frontend
# Feature/edit investment large capital investor details

## Description of change

Replace Investments - large capital investor details with react

## Test instructions

Navigate to https://www.datahub.dev.uktrade.io/companies/9efc3830-4d6e-4e04-96c7-5451e92363ee/investments/large-capital-profile?editing=investor-details where you should be able to edit a new or modify an existing investor-details to be able to 

## Screenshots
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/19926221/156989504-b43f6ad7-ebb4-4cf6-a06b-83463402c5b2.png">

_Add a screenshot_

<img width="1013" alt="image" src="https://user-images.githubusercontent.com/19926221/156989595-14d65821-737c-4134-93c2-7c6007082fb5.png">

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
